src/integration-tests/integration-test.py

```diff
@@ -140,6 +166,18 @@ class Integration(BaseModel):
             ContainerType.coverage: 1,
         },
     ),
+    "windows-libfuzzer-linked-library": Integration(
```

Comment:
Can we add a `windows-libfuzzer-loadlibrary` integration test, similar to the `-dlopen` one?

Reply:
Yes, though that's in progress.  I will add that as a follow-on PR.